Podverse/FiltersTableHeaderView.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  FiltersTableHeaderView.swift
+//  Podverse
+//
+//  Created by Creon Creonopoulos on 10/3/17.
+//  Copyright © 2017 Podverse LLC. All rights reserved.
+//
+
+import UIKit
+
+protocol FilterSelectionProtocol {
+    func filterButtonTapped()
+    func sortingButtonTapped()
+    func sortByTopHour()
+    func sortByTopDay()
+    func sortByTopWeek()
+    func sortByTopMonth()
+    func sortByTopYear()
+    func sortByTopAllTime()
+}
+
+class FiltersTableHeaderView: UIView {
+
+    var delegate:FilterSelectionProtocol?
+    var filterTitle = "" {
+        didSet {
+            DispatchQueue.main.async {
```

Comment:
Was there an issue and you decided to dispatch to main here?

Reply:
Yes, for some reason the didSet methods (filterTypeSelected didSet on the vc -> filterTitle didSet on the FiltersTableHeaderView) do not update the UI on viewDidLoad, unless I dispatch to the main queue here.

I'm not sure why :|

Reply:
👍